src/Games/brain-even.js

```diff
@@ -1,29 +1,10 @@
-import readlineSync from 'readline-sync';
 import getRandomNumber from '../randomCount.js';
```

Comment:
Don't use PascalCase for folder names, plz use camelCase.
PascalCase using for classes or for example React components

Reply:
Yep)